instrumentation/graphql-java-16.2/build.gradle

```diff
@@ -0,0 +1,31 @@
+dependencies {
+    implementation(project(":agent-bridge"))
+
+    implementation 'com.graphql-java:graphql-java:16.2'
```

Comment:
Do we need to add something to the `THIRD_PARTY_NOTICES.md` file?

Reply:
??? this is the library we are instrumenting.  I don't know about this type of file.

Reply:
Not for dependencies used in instrumentation modules.

Reply:
Typically it's just libraries that we directly use in development of the agent that need to be declared.